src/pages/InsideContents/settings/MyAccountPage.tsx

```
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
```

ESLint

[error] 'ThemeProvider' is defined but never used.

(@typescript-eslint/no-unused-vars)